### PR TITLE
Improve Student Projects Section

### LINKED ISSUE #446: Improve Student Projects Section
**Actual Behaviour**
Popovers are not looking good for the current usage (displaying images). 

**Screenshots of the issue**

image

**Would you like to work on the issue?**
Yes , I would like to work on the issue .

### PR DESCRIPTION
**Pull request by KSNINJA**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://ksninja.github.io/temp/students_projects.html)

- Fixes #446 

## Description / Changes
Replaced the previous functionality of popovers with flip animation to show snapshot of project.
Clicking on the text flips the card and displays the image .

### After:
image

- - - - - - - - - - - -

### CODE REVIEW

File: .~lock.students_projects.html# [hunk 1]
```diff
@@ -0,0 +1 @@
+Kartik Singhal,kartiksinghal,Kartiks-MacBook-Pro.local,22.11.2018 12:38,file:///Users/kartiksinghal/Library/Application%20Support/OpenOffice/4;
```
Comment (nikit19) [inline .~lock.students_projects.html#]: Why?

**Comment by pythongiant** [comment]
image
Center the blocks please?

**Comment by KSNINJA** [comment]
@nikit19 I made the changes . You can review now .  Also , @sahilsaha7773 made a comment . please confirm if those changes are necessary . I think this is good to flip all together .

**Comment by fragm3** [comment]
@KSNINJA link not working.

**Comment by KSNINJA** [comment]
Sorry about all the trouble . Actually , I am first time working with different branches . So , got a little confused . You can now review .

**Comment by KSNINJA** [comment]
I have made the changes . please review @fragm3

**Comment by KSNINJA** [comment]
I have made the changes . Pls review @fragm3

**Comment by KSNINJA** [comment]
These errors are related to other files , which I have not edited (footer.html  and thankyou.scss) .  
I could definitely tried that , but I am aware of what it is doing (there are not comments to explain code)  .

**Comment by sk9331657** [comment]
> These errors are related to other files , which I have not edited (footer.html and thankyou.scss) .
> I could definitely tried that , but I am aware of what it is doing (there are not comments to explain code) . These errors are in footer which is linked to students-projects.html
> image

Make sure your codebase is updated because  https://gci18.fossasia.org/ doesn't seem to have the error.

**Comment by KSNINJA** [comment]
image
No , gci18.fossasia.org also have the same errors

**Comment by KSNINJA** [comment]
gci18.fossasia.org also have the same errors . Even then , I updated my code with upstream .
image
@sk9331657

**Pull request merged**